notifications/notifications.go

```diff
@@ -0,0 +1,35 @@
+package notifications
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/asobti/kube-monkey/chaos"
+	"github.com/asobti/kube-monkey/config"
+	"github.com/golang/glog"
+)
+
+func Send(client Client, endpoint string, msg string, headers map[string]string) error {
+	if err := client.Request(endpoint, msg, headers); err != nil {
+		return fmt.Errorf("send request: %v", err)
+	}
+	return nil
+}
+
+func ReportAttack(client Client, result *chaos.Result, time time.Time) bool {
```

Comment:
I'm assuming that the `.attacks` part in the configuration is so that this can be extended with other types of notifications, such as `.schedule`, etc later on?

Do you mind adding a note (maybe in this file) as a reminder of this?